libc/cmake/modules/LLVMLibCHeaderRules.cmake

```diff
@@ -144,18 +243,18 @@ function(add_gen_header target_name)
     file(MAKE_DIRECTORY ${LIBC_INCLUDE_DIR}/llvm-libc-decls/gpu)
     set(decl_out_file ${LIBC_INCLUDE_DIR}/llvm-libc-decls/${relative_path})
     add_custom_command(
-      OUTPUT ${decl_out_file}
-      COMMAND ${hdrgen_exe} -o ${decl_out_file}
-              --header ${ADD_GEN_HDR_GEN_HDR} --def ${in_file} --export-decls
-              ${replacement_params} -I ${LIBC_SOURCE_DIR} ${ENTRYPOINT_NAME_LIST_ARG}
-              ${LIBC_SOURCE_DIR}/config/${LIBC_TARGET_OS}/api.td
+    OUTPUT ${decl_out_file}
```

Comment:
why is this changed?

Reply:
I'm not too sure, that was there from Aaryan's commit. The code that was removed/replaced looks the same though.

Reply:
Accidentally changed spacing when deleting and adding will make it the same again